ext/iconv/tests/iconv_strpos_error_001.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+iconv_strpos() - test against PHP's defaults internal encoding. Non windows systems.
+--EXTENSIONS--
+iconv
+--SKIPIF--
+<?php
+if (substr(PHP_OS, 0, 3) == "WIN" ) {
+    die("skip test is not for windows systems only");
+}
+?>
+--FILE--
+<?php
+$large_enconding = str_repeat("X", 256);
+ini_set("default_charset", $large_enconding);
+var_dump(iconv_strpos('Hello, world', 'world', -2));
+?>
+--EXPECTF--
+Warning: ini_set(): Unknown encoding %s
+
+Warning: ini_set(): INI setting contains invalid encoding %s
+
```

Comment:
It seems to me that the differences in the output do not depend on the operating system, but are rather related to whether mbstring is enabled; if it is, then the `php_internal_encoding_changed` hook is set, causing the additional warnings about the ini setting. Since these warnings are not particularly relevant to the test at hand, I think we should just ignore them:
```suggestion
%A
```

Reply:
@cmb69 thanks a lot!!! I really aporeciate this. Specially by the fact I Am starting this "contribution journey".

I Will change my tests and lets see how It goes

Reply:
I did the changes. By the way.... What   %A means? @cmb69

Reply:
`%A` means zero or more of anything (character or white space) including the end of line character. See http://qa.php.net/phpt_details.php#expectf_section for a list of supported format specifiers.